scrapper/server/db.py

```diff
@@ -0,0 +1,30 @@
+from flask_restful import Api
+from flask_pymongo import PyMongo
+import os
+from pymongo import errors
+
+
+class Db:
```

Comment:
some generic comment here:
Every method for database insert have `try/except` block 
My recommendation: Create extra private method for class to avoid copy/pasted try except for connection block

Other solution and this would be the best from my PoV: Verify if the client was successfully connected to database, and if connection cannot be established log warning/error (don't raise an error just log it so the console will show that database is inaccessible)